app/install/google-cloud.md

```diff
@@ -0,0 +1,97 @@
+---
+id: page-install-method gke-deployment
+title: Install - Google Cloud Platform
+header_title: Google Cloud Platform
+header_icon: /assets/images/icons/icn-installation.svg
+breadcrumbs:
+  Installation: /install
+links:
+  gke_repo: "https://github.com/Mashape/kong-distribution-gke"
+---
+
+Kong and its supported database can easily povisioned using the templates which can be dowloaded or cloned from following repo:
+
+- [Download GKE deployment templates]({{ page.links.gke_repo }})
+
+----
+
+### Notes:
+
+1. **Initial setup**:
+
+    You would need `gcloud` and `kubectl` command-line tools installed and setup to run deployment commands.
+
+    Download or clone the above mentioned repo
+
+    ```bash
+    $ git clone git@github.com:Mashape/kong-dist-google-cloud.git
+    $ cd kong-dist-google-cloud
+    ```
+
+2. **Deploy a cluster**:
+
+    Using `cluster.yaml`, deploy a GKE cluster to use for deploying the solution later. Fill in the following information before deploying:
+    
+    * desired cluster name
+    * zone in which to run the cluster
+    * basicauth username and password for authenticating access to the cluster
+
+    When ready, deploy with the following command:
+
+    ```bash
+    $ gcloud deployment-manager deployments create cluster --config cluster.yaml
+    ```
+
+3. **Deploy a Kong supported Database (currently only Cassandra supported)**
+
+    Using `cassandra.yaml` or `postgres.yaml`, deploy a Cassandra `Service` and a `ReplicationController` to the GKE cluster created in the last step.
```

Comment:
if only Cassandra is supported, why is there a `postgres.yaml`?

also, shouldn't the `.yaml` files be `.yml` ?